components/brave_sync/brave_profile_sync_service_impl.h

```diff
@@ -139,6 +143,7 @@ class BraveProfileSyncServiceImpl
   FRIEND_TEST_ALL_PREFIXES(::BraveSyncServiceTest, ClientOnGetInitData);
   FRIEND_TEST_ALL_PREFIXES(::BraveSyncServiceTest, OnSaveBookmarksBaseOrder);
   FRIEND_TEST_ALL_PREFIXES(::BraveSyncServiceTest, OnCompactComplete);
+  FRIEND_TEST_ALL_PREFIXES(::BraveSyncServiceTest, OnRecordsSent);
```

Comment:
The same as above.

Reply:
addressed in 73b682f8175917b4bd8e2356088acb0af9924217